Analyticsv3Sample/README.md

```diff
@@ -0,0 +1,50 @@
+
+## DAIMTO Google Analytics API Sample for .NET  ##
+
+View and manage your Google Analytics data
+
+## Description ##
+The DAIMTO Google Analytics API Sample for .NET instructs you in the usage of the Google APIs client library for .NET.
+
+The sample contains OAuth2.0 authentication, service account, and and public API.   As well as methods for each of the Resources avaliable in the Analytics API.  
+
+Tutorials for some of these sample projects can be found on my website [Daimto.com](http://www.daimto.com)
+
+Requirements 
+=================================
+* [.NET Framework 4.0](http://www.microsoft.com/en-us/download/details.aspx?id=17851)
+* [.NET Framework 4.5](http://www.microsoft.com/en-us/download/details.aspx?id=30653)
+* [Nuget package Google Apis Analytics v3](https://www.nuget.org/packages/Google.Apis.analytics.v3/)
+  *   PM> Install-Package Google.Apis.Analytics.v3
+
+SETUP
+=================================
+
+Make sure you create your own Oauth credentials at https://console.developers.google.com/.
+
+Walk though on how to create the diffrent credentails can be found:
+* [Google Developer console service account](http://www.daimto.com/google-developer-console-service-account/)
+* [Google Developer Console Create Public API Key](http://www.daimto.com/google-developer-console-create-public-api-key/)
+* [Google Developer Console Oauth2 credentials](http://www.daimto.com/google-developer-console-oauth2/)
+
+Projects where created in visual studio 2013.  
+
+Developer Documentation
+=================================
+
+* [Google API client Library for .NET - Get Started](https://developers.google.com/api-client-library/dotnet/get_started)
+* [Analytics v3 Documentation](https://developers.google.com/analytics/)
+    
+
+Source Code
+=================================
+
+The source code for the Google APIs .Net client library can be found at [Google APIs Client Library for .NET](https://github.com/google/google-api-dotnet-client)
+The source code for the Google Analytics API csharp can be found at [Analytics v3 csharp API](https://developers.google.com/resources/api-libraries/download/analytics/v3/csharp).
+
+Support Forums
+=================================
+If you find an issue with in the sample project I would appricate you adding an issue in the [issue tracker](https://github.com/LindaLawton/Google-Dotnet-Samples).  
+Please supply the name of the sample you are repporting an issue for in the title. Example: \[Analytics Sample\] Doesnt work.
```

Comment:
Maybe change to:

Please include the name of the sample in the issue title. Example...